src/main/drivers/max7456_symbols.h

```diff
@@ -214,7 +214,7 @@
 
 // Throttle Position (%)
 #define SYM_THR   0x04
-#define SYM_THR1  0x04
+#define SYM_THR1  0x20
```

Comment:
what is this change for? I thought we fixed the throttle osd issue?

Reply:
`git checkout master -- src/main/drivers/max7456_symbols.h` and re-commit